src/MSBuild/Resources/Strings.resx

```diff
@@ -83,7 +83,8 @@
     </comment>
   </data>
   <data name="CannotAutoDisableAutoResponseFile" UESanitized="false" Visibility="Public">
-    <value>MSBUILD : error MSB1027: The -noAutoResponse switch cannot be specified in the MSBuild.rsp auto-response file, nor in any response file that is referenced by the auto-response file.</value>
+    <value>MSBUILD : error MSB1027: The -noAutoResponse switch cannot be specified in the MSBuild.rsp auto-response file, nor in any response file that is referenced by the auto-response file.
+    Full command line: {0}</value>
```

Comment:
I think it's unfortunate that so many resource strings are changed and will need attention from the localization team. Do you think the command line could be printed using only one shared resource instead?